packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -360,23 +360,36 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
-export const llama_cpp_python = (model: ModelData): string[] => [
-	`from llama_cpp import Llama
+export const llama_cpp_python = (model: ModelData): string[] => {
+	let snippet = `from llama_cpp import Llama
```

Comment:
switching to an array might be more legible

Reply:
handled in https://github.com/huggingface/huggingface.js/pull/954/commits/b9baabc2cfd5ff28deddebf5bd0e29efc95e9d97